tutorindigo/plugin.py

```diff
@@ -55,3 +60,15 @@
     [(f"INDIGO_{key}", value) for key, value in config["unique"].items()]
 )
 hooks.Filters.CONFIG_OVERRIDES.add_items(list(config["overrides"].items()))
+
+
+hooks.Filters.ENV_PATCHES.add_item(
+    (
+        "mfe-dockerfile-post-npm-install-learning",
+        """
+RUN npm install '@edx/brand@git+https://github.com/edly-io/brand-openedx.git#indigo'
+RUN npm install '@edx/frontend-component-header@git+https://github.com/edly-io/frontend-component-header.git#indigo'
+RUN npm install '@edx/frontend-component-footer@git+https://github.com/edly-io/frontend-component-footer.git#indigo'
```

Comment:
These packages should refer to a release-specific branch, such that past releases can still be built in the future. E.g: "@edx/frontend-component-footer@git+https://github.com/edly-io/frontend-component-footer.git#indigo/quince"

Reply:
I got your point. Regis, we are not moving forwardd with release or release-specific branches. Because when I build the image using the `git` urls as now, our learning mfe has generated errors on `npm run build`. As it requires files from header and footer `build` folder which is not existing in `git` urls/repos. So, decided to publish the packages of header and footer and provides their npm packages name instead of `git` urls.

Regarding the release-specific branch, I'll create that. Thanks for the point